### ISSUE TITLE
`send` function does not accept same parameter-types in `v0.0.2` as it does in `v0.0.1`

### ISSUE THREAD

**Issue by chazkiker2** [Light-Labs/ryderserial-proto#9]
# Description

ryderserial-proto `v0.0.2` does not accept the same parameter types as it did in `v0.0.1`. 

I noticed this and then apparently forgot about it during the initial PR... but this is a super breaking change. 

Take the following code, for example: 

```javascript
var cmd = new Uint8Array(2);
cmd[0] = RyderSerial.COMMAND_EXPORT_PUBLIC_IDENTITY;
cmd[1] = identity_number;
var identity = await ryder_serial.send(cmd);
```

This is a super common implementation of the `send` function, but the `send` function no longer accepts `Uint8Array` or `Array<number>` or `Array<string>`.

**Comment by MarvinJanssen** [comment]
Can you hotfix this? I didn't notice. Adding you as a contributor so you can merge (squash please).

We need unit tests hehe.

**Comment by chazkiker2** [comment]
bet. Will hotfix today. Big miss on my part... so sorry.

**Comment by chazkiker2** [comment]
@MarvinJanssen this may require a refactor of the `Train[0]` type...

**Comment by chazkiker2** [comment]
shit! send also used to return `Promise<string | ???>` based on the `export.js` command in `ryder-cli-proto`...

<details>

<summary>Example code</summary>

```typescript
    const { args, flags } = this.parse(Export)
    if (!this.ryder_serial) {
      return;
    }
    let commands: number[] = [];
    if (args.what === 'identity')
      commands.push(RyderSerial.COMMAND_EXPORT_PUBLIC_IDENTITY);
    else {
      // TODO: RyderSerial.COMMAND_EXPORT_OWNER_APP_KEY_PRIVATE_KEY
      if (args.what === 'owner_key') {
        commands.push(flags.private_key ? RyderSerial.COMMAND_EXPORT_OWNER_KEY_PRIVATE_KEY : RyderSerial.COMMAND_EXPORT_OWNER_KEY);
      } else { // app_key
        commands.push(flags.private_key ? RyderSerial.COMMAND_EXPORT_APP_KEY_PRIVATE_KEY : RyderSerial.COMMAND_EXPORT_APP_KEY);
      }
    }
    commands.push(args.id_number);
    let response = await this.ryder_serial.send(commands);
    if (args.what !== 'app_key') {
      console.log(response);
    }
    else if (response !== RyderSerial.RESPONSE_SEND_INPUT) {
      console.log('Ryder did not request user input, it might doing something else or it is stuck in the wrong mode.');
    }
    else {
      response = await this.ryder_serial.send(args.app_domain+"\0");
      if (response !== RyderSerial.RESPONSE_REJECTED) {
        console.log(response);
      }
    }
    this.ryder_serial.close();
```
</details>

**Comment by chazkiker2** [comment]
fixed in PR #10
